proto/pom.xml

```diff
@@ -44,6 +44,12 @@
             <groupId>com.google.protobuf</groupId>
             <artifactId>protobuf-java</artifactId>
         </dependency>
+
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>22.0</version>
+        </dependency>
     </dependencies>
```

Comment:
Can you add this to bom-dependancies dependencyManagement [1] block please, making sure its in alphabetical order. You don't need to have the version specified here then - just like how the plugins work.

[1] https://github.com/DozerMapper/dozer/blob/master/bom-dependencies/pom.xml#L58